### PR TITLE
✏ Update Hypercorn link, now pointing to GitHub

### PR DESCRIPTION
**Pull request by baconfield**
Updated Uvicorn link from gitlab to github to reflect current hosting
https://gitlab.com/pgjones/hypercorn/-/commit/65e539a9dc762cb19f4907038fe6d490dac909ff

### CODE REVIEW

File: docs/en/docs/index.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ $ pip install fastapi
 
 </div>
 
-You will also need an ASGI server, for production such as <a href="https://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a> or <a href="https://gitlab.com/pgjones/hypercorn" class="external-link" target="_blank">Hypercorn</a>.
+You will also need an ASGI server, for production such as <a href="https://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a> or <a href="https://github.com/pgjones/hypercorn" class="external-link" target="_blank">Hypercorn</a>.
```
Comment (iudeen) [inline docs/en/docs/index.md:131]: I think it should be https://hypercorn.readthedocs.io/

**Review by iudeen — CHANGES_REQUESTED** [review]
The title of the PR should be "Update Hypercorn link" not Uvicorn....

File: docs/en/docs/index.md (hunk 1 above)
Comment (iudeen) [inline docs/en/docs/index.md:131]:
```suggestion
You will also need an ASGI server, for production such as <a href="https://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a> or <a href="https://hypercorn.readthedocs.io/" class="external-link" target="_blank">Hypercorn</a>.
```

**Review by iudeen — CHANGES_REQUESTED** [review]

**Comment by baconfield** [comment]
> The title of the PR should be "Update Hypercorn link" not Uvicorn....

Sorry about that, was half asleep there

**Comment by tiangolo** [comment]
Nice, thanks @baconfield! :cake: 

And thanks for the help @iudeen!

**Pull request merged**